filament/src/details/VertexBuffer.cpp

```diff
@@ -283,55 +283,49 @@ FVertexBuffer::FVertexBuffer(FEngine& engine, const Builder& builder)
     mHandle = driver.createVertexBuffer(mVertexCount, mVertexBufferInfoHandle,
             utils::ImmutableCString{ builder.getName() });
 
+    auto shouldCreateBuffer = [this](size_t attributeIndex) {
+        if (mBufferObjectsEnabled) {
+            if (!mAdvancedSkinningEnabled) {
```

Comment:
Looks like this case can be entirely taken outside the loop. I think that will make it even easier to read.